app/views/devise/sessions/_mfa_form.erb

```diff
@@ -4,6 +4,6 @@
 
 <div class="govuk-form-group">
   <%= f.label :totp_code, class: "govuk-label" %>
-  <%= f.text_field :totp_code, autofocus: true, class: "govuk-input govuk-input--width-10" %>
+  <%= f.text_field :totp_code, autofocus: true, html: { autocomplete: "off" }, class: "govuk-input govuk-input--width-10" %>
```

Comment:
Wouldn't just this work?
```suggestion
  <%= f.text_field :totp_code, autofocus: true, autocomplete: 'off', class: "govuk-input govuk-input--width-10" %>
```

Reply:
I didn't try that way... That was the documentation I found... I shall try that way and come back to you

Reply:
OK that works... have made the change and repushed